### DISCUSSION TITLE
Listen Changes to CSV file

### DISCUSSION THREAD

**Discussion by tahaaslam1** [supabase/supabase#9259] [Questions]
After importing a CSV file to the Supabase database, is there a way we can listen to the changes made to the same CSV file?

**Comment by GaryAustin1** [comment]
Can you clarify what you are asking for?  When you import the CSV file it is usually into a table.  Once in that table you can use realtime to listen for changes.

**Comment by Sukikiroi** [comment]
You can do a script with node js or python to convert csv file to Array and then compare that file to the previos Array and then make a decision to insert or delete or update in the Supabase Table 

Or you can just make streaming from that file to the Supabase Table
